tests/vse_sync_pp/analyzers/test_phc2sys.py

```diff
@@ -0,0 +1,207 @@
+### SPDX-License-Identifier: GPL-2.0-or-later
+
+"""Test cases for vse_sync_pp.analyzers.ts2phc"""
+
+from unittest import TestCase
+from collections import namedtuple
+from decimal import Decimal
+import math
+
+from vse_sync_pp.analyzers.phc2sys import (
+    TimeErrorAnalyzer
+)
```

Comment:
```suggestion
from vse_sync_pp.analyzers.phc2sys import TimeErrorAnalyzer
```

Reply:
Alternatively, add a comma after `TimeErrorAnalyzer` if you're listing symbols out